plugins/woocommerce/tests/e2e/specs/smoke-tests/upload-plugin.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Internal dependencies
+ */
+const { merchant } = require( '@woocommerce/e2e-utils' );
+
+const { getRemotePluginZip, getLatestReleaseZipUrl, deleteDownloadedPluginFiles } = require( '@woocommerce/e2e-environment' );
+
+/**
+ * External dependencies
+ */
+const {
+	it,
+	beforeAll,
+} = require( '@jest/globals' );
+
+const { GITHUB_REPOSITORY, PLUGIN_NAME, GITHUB_TOKEN } = process.env;
+
+let zipUrl;
+let pluginPath;
+
+describe( 'Upload and activate plugin', () => {
```

Comment:
This should probably be a `describeIf` on `GITHUB_REPOSITORY` being defined.

Reply:
Good catch! Making that change now 👍